src/main/resources/scripts/plugin_testing.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+## Script to calculate the average performance of creating detector api
+count=$1
+for p in $*;
+do
+echo "------------"
+let i=$count-1
+tot=0
+while [ $i -ge 0 ];
+do
+res=`curl -w "$i: %{time_total} %{http_code} %{size_download} %{url_effective}\n" --location --request POST 'http://localhost:5601/api/anomaly_detectors/detectors' \
+--header 'Cookie: security_authentication=Fe26.2**8d2be269f3e27e62b52420bb4658d58d987c099e2753c43c44f50449b5196414*LdPlLCuNd4rqMTCevRNHMQ*hjgYZ629nnIIKNw3r31KlMbCIS9azZKhuWsGHEYJOlbaZs3eOwtdslKMeIUR1KylieB39cxByqXEpeqSmrtLp6VLZABWGa-b4hO3bW11VSp_OyOWnORk7lDSHedbUplbxtKMaTXgIsJFT8gNvNBM6DeNEPqk8V2naYcW43lbslj3cE2PoSHwlMekguHQSojXFA2x-paKnEgPOiiXsoX9EFrkYLj_ZlnHyjkz9YZKsew**032e9b7b5965967558c0c31b0b94301df82b2111159de5db435f954e86699091*NnIB4AHDYcnhBoG085xZ5lYGD_E0dExvFSXqdw2FYcs' \
```

Comment:
Can we not expose this cookie?

Reply:
yeah removed it. Anyways it is common architecture for any api to be tested. Also this cookie expires. It is just for the reference I added.